### PR TITLE
Import Business Central Attributes API added

### PR DESCRIPTION
**Pull request by ruuushhh**

### CODE REVIEW

File: apps/business_central/helpers.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    Synchronize various dimensions with Business Central using the provided credentials.
+
+    :param business_central_credential: BusinessCentralCredentials Instance
+    :param workspace_id: ID of the workspace
+
+    This function syncs dimensions like accounts, vendors, commitments, jobs, categories, and cost codes.
+    """
+
+    # Initialize the Business Central connection using the provided credentials and workspace ID
+    business_central_connection = import_string('apps.business_central.utils.BusinessCentralConnector')(business_central_credential, workspace_id)
```
Comment (NileshPant1999) [inline apps/business_central/helpers.py:49]: check the spelling of business

File: apps/business_central/helpers.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    # List of dimensions to sync
+    dimensions = ['accounts', 'vendors', 'employees', 'locations']
```
Comment (NileshPant1999) [inline apps/business_central/helpers.py:52]: are these it ??
i mean jobs and other are not there

File: apps/business_central/views.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import logging
+
+from rest_framework import generics
+
+from apps.business_central.serializers import ImportBusinessCentralAttributesSerializer
+
+
+logger = logging.getLogger(__name__)
+logger.level = logging.INFO
```
Comment (NileshPant1999) [inline apps/business_central/views.py:9]: i dont think it is used anywhere

**Review by NileshPant1999 — APPROVED** [review]

File: apps/business_central/helpers.py (hunk 2 above)
Comment (ruuushhh) [reply apps/business_central/helpers.py:52]: yes this is it
Will confirm with Shwetabh tough once he is back

File: apps/business_central/views.py (hunk 3 above)
Comment (ruuushhh) [reply apps/business_central/views.py:9]: will remove

Comment (ruuushhh) [reply apps/business_central/views.py:9]: fixed this in https://github.com/fylein/fyle-ms-business-central-api/pull/29

**Pull request merged**